src/stylesheets/components/_nav-header.scss

```diff
@@ -6,7 +6,8 @@
   margin-bottom: $padding-xxxlarge;
   margin-top: $padding-large;
 
-  .btn-nav-header {
+  button {
+    font-size: $font-medium;
```

Comment:
A better name for this variable would be `$font-size-medium`. At first sight, I thought "$font-medium" was referring to the font-weight.